addons/mail/models/ir_attachment.py

```diff
@@ -76,6 +77,9 @@ def _delete_and_notify(self, message=None):
             )
         self.unlink()
 
+    def _get_store_attachment_ownership_fields(self):
+        return [Store.Attr("attachment_ownership", lambda a: a._attachment_ownership_access_token())]
```

Comment:
`_token`? not necessarily clear what this is otherwise when just looking at js :shrug: